packages/flutter/test/widgets/semantics_role_checks_test.dart

```diff
@@ -189,40 +188,6 @@ void main() {
       expect(error.message, 'Radio groups must not have multiple checked children');
     });
 
-    testWidgets('error case, reports first error', (WidgetTester tester) async {
```

Comment:
I don't know how to properly test this since now only multiple checked radio will throw error, and since this error involves all radios in the group, so there  it doesn't make sense to say one of the error is the `first error`